main_test.go

```diff
@@ -49,3 +49,7 @@ func TestMain(m *testing.M) {
 
 	os.Exit(ret)
 }
+
+func createEmptyFile(path string) (err error) {
```

Comment:
I guess it could. Whether we want to do that now or maybe later when we have a few other functions to put in the file I don't know?